engine/chainstate/interface.go

```diff
@@ -21,9 +21,6 @@ type ChainService interface {
 	QueryTransaction(
 		ctx context.Context, id string, requiredIn RequiredIn, timeout time.Duration,
 	) (*TransactionInfo, error)
-	QueryTransactionFastest(
```

Comment:
Just out of curiosity. Why is this method unused and obsolete now? Was it inefficient or sth else?

Reply:
The `chainstate` was designed to use several ARC/mAPI services.
- But now we use only one ARC service - so the majority of the logic here is unnecessary.

Besides - the package will be (most probably) "zaorany" soon :D

Reply:
it doesn't do what it's state that it's doing, because it will just use the go-broadcast-client which will try to query one by one.